packages/react-core/src/components/DescriptionList/__tests__/DescriptionListDescription.test.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { DescriptionListDescription } from '../DescriptionListDescription';
+
+import styles from '@patternfly/react-styles/css/components/DescriptionList/description-list';
+
+test('Renders to match snapshot', () => {
+  const { asFragment } = render(<DescriptionListDescription>test</DescriptionListDescription>);
+  expect(asFragment()).toMatchSnapshot();
+});
+
+test(`Renders default class ${styles.descriptionListDescription}`, () => {
+  render(<DescriptionListDescription>test</DescriptionListDescription>);
+  expect(screen.getByText('test').parentElement).toHaveClass(styles.descriptionListDescription);
```

Comment:
```suggestion
  expect(screen.getByText('test').parentElement).toHaveClass(styles.descriptionListDescription, {exact: true});
```

Also a test that spread props works correctly